inspirehep/config.py

```diff
@@ -1452,6 +1452,7 @@ def _(x):
             "workflow_name": terms_filter('_workflow.workflow_name'),
             "is-update": terms_filter('_extra_data.is-update'),
             'subject': terms_filter('metadata.inspire_categories.term'),
+            "decision": terms_filter('_extra_data.relevance_prediction.decision')
```

Comment:
Nit: add an extra comma to the last element. If you don't do it, when you append again you get a slightly messier diff, where it looks like you deleted the previous line, while all you did was adding the extra comma.